test.sh.in

```diff
@@ -266,34 +269,105 @@ waitForAPI() {
   fi
 }
 
+######################################
+#    OVA POST SMOKE TEST RELATED     #
+######################################
+portForwarding(){
```

Comment:
as we talked, I'm fine with the port forwarding in your PR if you have well tested.
but in the long run, I still insist to use as simple way as possbile to setup an OVA network.